CHANGELOG.md

```diff
@@ -3,9 +3,89 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 4.0.0-alpha.38
+
+_Aug 23, 2021_
+
+This is the first offitial stable release of the `DataGrid` and `DataGridPro`. It supports `@material-ui/core` v4 and has a partial support for v5. From this release on we will release new features for the `DataGrid` and `DataGridPro` under version `5.0.0-beta.0` and have full support for `@material-ui/core` v5.
```

Comment:
@oliviertassinari I need some inspiration regarding this freestyle text 😅

Reply:
I have a few ideas on how to expend it

Reply:
Updated